### DISCUSSION TITLE
Cannot activate my copilot in Rstudio

### DISCUSSION THREAD

**Discussion by SophiaSun123** [community/community#140200] [GitHub Education]
### Hi everyone,

Hey, my request gets approved and I also received the confirmation email. However, when I sign in my account in Rstudio, it said that I was not activated. I do not know what to do. Is there any activation code I need to use? Thanks!

**Comment by SophiaSun123** [comment]
Problem solved! You just need to click on the "Free Trial"

**Reply by AESMatias** [reply]
Sorry, i had this page opened, i did not realize that was already solved lol

**Comment by AESMatias** [comment]
Hello @SophiaSun123, did you installed and enabled copilot in the global options > GitHub ? 
Also, check you're signed in with your copilot linked account, try closing and open Rstudio again.
